### PR TITLE
Full adoption of WebDriverManager

### PR DESCRIPTION
**Pull request by jedstromcsv**
This PR moves the WebDriverManager into a fully adopted dependency.

### CODE REVIEW

File: jmaqs-selenium/src/main/java/com/magenic/jmaqs/selenium/WebDriverFactory.java [hunk 1]
```diff
@@ -197,6 +195,7 @@ public static WebDriver getChromeDriver(ChromeOptions chromeOptions) {
    * @return the chrome driver
    */
   public static WebDriver getChromeDriver(ChromeOptions chromeOptions, String size) {
+    WebDriverManager.chromedriver().setup();
```
Comment (twalshcsv) [inline jmaqs-selenium/src/main/java/com/magenic/jmaqs/selenium/WebDriverFactory.java:198]: Would it be possible, reasonable to add this as something like a lazy initializer.
This should only need to run once.

File: jmaqs-selenium/src/main/java/com/magenic/jmaqs/selenium/WebDriverFactory.java [hunk 2]
```diff
@@ -7,12 +7,10 @@
 import com.magenic.jmaqs.selenium.constants.BrowserType;
 import com.magenic.jmaqs.selenium.constants.OperatingSystem;
 import com.magenic.jmaqs.selenium.constants.RemoteBrowserType;
-import com.magenic.jmaqs.selenium.constants.WebDriverFile;
 import com.magenic.jmaqs.selenium.exceptions.DriverNotFoundException;
 import com.magenic.jmaqs.selenium.exceptions.WebDriverFactoryException;
-import com.magenic.jmaqs.utilities.helper.Config;
-import com.magenic.jmaqs.utilities.helper.ConfigSection;
 import com.magenic.jmaqs.utilities.helper.StringProcessor;
+import io.github.bonigarcia.wdm.WebDriverManager;
```
Comment (twalshcsv) [inline jmaqs-selenium/src/main/java/com/magenic/jmaqs/selenium/WebDriverFactory.java:13]: We should have a fall back in case WebDriverManager functionality is blocked.
This could easily be put in a different PR.

**Review by twalshcsv — APPROVED** [review]
Small comments - nothing that would stop this from getting in

File: jmaqs-selenium/src/main/java/com/magenic/jmaqs/selenium/WebDriverFactory.java (hunk 1 above)
Comment (jedstromcsv) [reply jmaqs-selenium/src/main/java/com/magenic/jmaqs/selenium/WebDriverFactory.java:198]: I'll do a quick double check.  The WDM should look at the local cache first unless we set `wdm.forceDownload=true`.
